src/components/Result.tsx

```diff
@@ -53,8 +55,29 @@ function generateCreatorElements(item: CMMStudy) {
   return creators;
 }
 
+// Randomize array using Durstenfeld shuffle algorithm
+function shuffleArray(array: Array<any>) {
```

Comment:
This should be generic, otherwise the type of the array is erased on return

```suggestion
function shuffleArray<T>(array: Array<T>) {
```